packages/v4/src/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -0,0 +1,104 @@
+---
+id: Time picker
+section: components
+related: ["Calendar month", "Date picker"]
+---
+
+A **time picker** allows users to select a time from a list of options. Time can be displayed in hours and minutes, but not in seconds.
+
+## Elements
+
+<img src="./img/Time picker- elements.png" alt="Time picker elements" width="362"/>
+
+1. **Time field**: Displays a user's selected time. Always fill the time field with a placeholder time format (HH:MM) or a pre-selected time. Users can choose a time by either typing in a time, or selecting one from the menu list of time options. The time field should never be empty.
+2. **Time field in focused state**: Indicates that menu list of time options is open.
+3. **Menu**: Contains a list of times in set increments, such as 12:00 AM to 11:30 PM in 30 minute increments.
+4. **Menu item in hover state**
+5. **Menu item in disabled state**
+
+## Usage
+Use time pickers to enable users to select or input a time. For example, in a toolbar for item filtration, or in a form for scheduling inputs.
+
+### Single time selection
+A time picker allows users to select a single time. When a user enters a new time in the time field, the new time automatically replaces any placeholder or previously-selected value in that input field.
+
+Typically, time picker menus list selectable times in 30-minute increments. However, users always have the option to input any desired times that don't fall on the suggested time increments. For example, a user could type in 12:38 PM despite the list of options only showing 12:30 PM and 1:00 PM options.
+
+For guidance on writing time formats for time pickers, see the [content](#time-format) section.
+
+
+### Time range selection
+Two time pickers can be used together to specify a time range. The first one functions as a start (“from”) time and the second one functions as an end (“to”) time. 
+
+<img src="./img/time range.png" alt="Time picker elements" width="300"/>
+
+Until the user selects a start time, the end time field will be disabled. 
+
+<img src="./img/Time range dropdown.png" alt="Time picker elements" width="302"/>
+
+### Date and time range selection
+Combine a date picker and time picker when you want users to select a range of times across multiple days. 
+
+For more information about using date and time pickers together, visit our [date picker guidelines](/components/date-picker).
+
+## Variations
+### In a toolbar
+Use time pickers as filters in a toolbar. Note that time pickers aren't commonly used on their own, and will usually be accompanied by a [date picker](/components/date-picker)
+
+See the [date picker usage guidelines](/components/date-picker)  to learn more about using date pickers and time pickers in a toolbar. 
```

Comment:
would link sepcifically to design guidelines tab
so add **/design-guidelines** at the end of the URL!